src/routes/auth/magic-link.ts

```diff
@@ -70,8 +69,7 @@ async function magicLink({ query }: Request, res: Response): Promise<unknown> {
     email: account.email,
     avatar_url: account.user.avatar_url
   }
-  const session: Session = { jwt_token, jwt_expires_in, user }
-  if (!useCookie) session.refresh_token = refresh_token
+  const session: Session = { jwt_token, jwt_expires_in, user, refresh_token, permission_variables }
```

Comment:
I don't think we should return `permission_variables` here (or any other place) to the client.